services/ui-src/src/types/states.ts

```diff
@@ -22,14 +23,14 @@ export interface AdminBannerState {
   bannerData: AdminBannerData | undefined;
   bannerActive: boolean;
   bannerLoading: boolean;
-  bannerErrorMessage: string;
+  bannerErrorMessage: ErrorVerbiage;
```

Comment:
We can make this a union type like `bannerData` and allow `undefined` as an option, so we can set the error state to a falsy value...